services/address-lookup/src/dev.js

```diff
@@ -0,0 +1,10 @@
+const findAddressListByPostcode = require("./services/findAddressListByPostcode");
+
+// npm run dev {apiKey}
```

Comment:
Whats the point of this?

Reply:
it's just something I used to develop the function and I thought it made sense to leave it in case something needs to be changed in the function.
you run `npm run dev abc123abc123` and it will return the response of function. it's different from the tests because tests assume the function is done and working as expected, while this is something that you want you use while you're developing it.

does not affect the functionality whatsoever and if you think it's not a good idea to leave this here, i can remove it

Reply:
yeah - leave it in in case this needs to be modified when other teams use it